spec/features/deploy_spec.rb

```diff
@@ -68,4 +68,25 @@
     expect_any_instance_of(Octokit::Client).to receive(:add_assignees).with('artsy/candela', 42, 'jane')
     DeployService.new(strategy).start
   end
+
+  it 'adds assignees to existing deploy PRs when unassigned' do
+    expect_any_instance_of(Octokit::Client).to receive(:create_pull_request)
+      .with('artsy/candela', 'release', 'staging', anything, anything)
+      .and_raise(Octokit::UnprocessableEntity)
+    allow_any_instance_of(Octokit::Client).to receive(:pull_requests)
+      .with('artsy/candela', base: 'release', head: 'staging')
+      .and_return([double(number: 42, assignee: nil)])
+    allow_any_instance_of(Octokit::Client).to receive(:pull_request_commits)
```

Comment:
#minor. Maybe we can always use `expect_...` to be stricter?

Reply:
I didn't want to concern this spec too much with _how_ the assignee is chosen, only that there is one.